recipe/meta.yaml

```
@@ -33,7 +33,11 @@ outputs:
         - libgcc-ng
       run_exports:
         strong:
+{% if cuda_major_minor < (11, 2) %}
           - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
+{% else %}
+          - cudatoolkit >={{ cuda_compiler_version }},<{{ cuda_major + 1 }}
```

Shouldn't we spell 11.2 out explicitly? What if `cuda_compiler_version` is 11.3? Or, based on the current plan, we don't release nvcc for 11.3+?
```suggestion
          - cudatoolkit >=11.2,<{{ cuda_major + 1 }}
```

---

No because if we build with a newer `cudatoolkit` version the package may pull in newer features (symbols) that would be unavailable in previous versions. So we would still want to set the minimum to be the version that we built with